force-app/main/default/classes/ComparisonResult.cls

```diff
@@ -0,0 +1,5 @@
+public class ComparisonResult {
```

Comment:
We should remove this class since it is not being used

Reply:
Yep, I will remove this, forgot to delete it.